auth-web/src/routes/router.ts

```diff
@@ -111,6 +112,11 @@ function mapPendingDetails (route: any) {
   }
 }
 
+function isStaff () {
+  const kcUserProfile = KeyCloakService.getUserInfo()
+  return kcUserProfile.roles.includes(Role.Staff)
```

Comment:
Do these statements always execute correctly or is there a need for a try-catch or conditional chaining?

Reply:
I haven't encountered an issue yet since this method is called as part of an authenticated 'only' route but the optional chaining would be wise, good call!
Will update

Reply:
If the called code is safe then nothing else is needed. Maybe just have a look that it can't throw an error or return a non-array.

Reply:
Definitely, will do.
With the added optional chaining in the event of a failed request on `KeyCloakService` the route would resolve to the default Dashboard Breadcrumb.